controller/helper/process-compliance-count.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+/**
+ * @module controller/handler/trial
+ */
+
+const database = require('../../model');
+const processPatient = require('../helper/process-patient');
+const processTrial = require('../helper/process-trial');
+const moment = require('moment');
+
+function getCount(row)
+{
+    var redCount = 0;
+    var yellowCount = 0;
+    var greenCount = 0;
+
+    console.log("prabhanjan");
+    //console.log(len(row));
+    var firstrow = null;
+    for (let item of row)
+    {
+        console.log("here");
+        firstrow = item;
+    }
+
+    for(var i=0; i < firstrow.length; i++)
```

Comment:
when dealing with arrays, unless you need direct access to the index, please use the `for of` loop over the plain `for` loop